frequency-plans.yml

```
@@ -198,16 +381,33 @@
   base-frequency: 915
   file: AS_923_2.yml
 
+- id: AS_923_2_TTN
+  band-id: AS_923_2
+  name: Asia 920-923 MHz (AS923 Group 2) (used by TTN)
+  description: TTN Community Network frequency plan for Asia 920-923 MHz (AS923 Group 2)
+  base-frequency: 915
+  base-id: AS_923_2
+  file: AS_923_2_TTN.yml
+
+- id: AS_923_2_TTN_VN
+  band-id: AS_923_2
+  name: Asia 920-923 MHz (AS923 Group 2) (used by TTN Việt Nam)
+  description: TTN Community Network frequency plan for Asia 920-923 MHz (AS923 Group 2) in Việt Nam
```

```suggestion
  description: TTN Community Network frequency plan for Asia 920-923 MHz (AS923 Group 2) in Vietnam
```

Let's stick to US English names otherwise this becomes a huge mess here

---

Fixed